### PR TITLE
master-qa-65267 

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
https://issues.liferay.com/browse/LRQA-65267

cc: @xbrianlee

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:bundle

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 1]
```diff
@@ -1522,4 +1522,77 @@ definition {
...
+		property test.name.skip.portal.instance = "SAML#ViewSPInitiatedSSOWithTransientNameId";
+		property portal.upstream = "gabor";
+
+		ApplicationsMenu.gotoPortlet(
+			category = "System",
+			panel = "Control Panel",
+			portlet = "Virtual Instances");
+
+		PortalInstances.addCP(
+			mailDomain = "liferay.com",
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1537]: `mailDomain` should be set to `www.able.com` too

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 2]
```diff
@@ -1522,4 +1522,77 @@ definition {
...
+uuid''',
+			nameIdentifierAttributeName = "static:CustomNameIDValue",
+			nameIdentifierFormat = "Transient",
+			samlEntityID = "samlsp",
+			idpAttributes = '''screenName=screenName
+emailAddress=emailAddress
+firstName=firstName
+lastName=lastName
+uuid=uuid''',
+			spURL = "http://www.able.com:8080");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1563]: If order doesn't matter, sort the values of each variable and sort the order of the variables

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 3]
```diff
@@ -1522,4 +1522,77 @@ definition {
...
+		User.logoutPG();
+
+		User.viewLoggedOutPG();
+
+		// Initiate Single Sign-On via SP
+
+		SAML.executeSPInitiatedSSO(
+			password = "test",
+			specificURL = "http://www.able.com:8080",
+			spURL = "http://www.able.com:8080",
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1574]: Where is this variable being used in `SAML.executeSPInitiatedSSO`?

**Comment by xbrianlee** [comment]
@gaborlovasdotcom This test exercises SSO with a transient ID, but we still need to cover the other part of LPS-125272 - asserting the user doesn't get created again on subsequent logins. Are you planning to cover that in a separate test or this one?

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro [hunk 4]
```diff
@@ -1,5 +1,138 @@
...
+		ApplicationsMenu.gotoPortlet(
+			category = "System",
+			panel = "Control Panel",
+			portlet = "Virtual Instances");
+
+		PortalInstances.addCP(
+			mailDomain = "${mailDomain}",
+			virtualHost = "${virtualHost}",
+			webId = "${webId}");
+	}
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro:49]: This macro isn't necessary. Also it's not specific to SAML so it shouldn't be in this file.

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro]: I remove this macro, and put these steps into the test case, thanks!

**Comment by gaborlovasdotcom** [comment]
Continued in https://github.com/xbrianlee/liferay-portal/pull/573

**Pull request closed without merging**